project-files/XML/DCC-signed.xml

```diff
@@ -1,567 +1,567 @@
urePolicyIdentifier></xades:SignedSignatureProperties><xades:SignedDataObjectProperties><xades:DataObjectFormat ObjectReference="#r-id-e17196bfef4de1755afb64bc0f392cac-1"><xades:MimeType>text/xml</xades:MimeType></xades:DataObjectFormat></xades:SignedDataObjectProperties></xades:SignedProperties><xades:UnsignedProperties><xades:UnsignedSignatureProperties><xades:SignatureTimeStamp Id="TS-6ddcc960-6535-4ae5-8d4b-a80446c608ff"><ds:CanonicalizationMethod Algorithm="http://www.w3.org/2001/10/xml-exc-c14n#"/><xades:EncapsulatedTimeStamp Id="ETS-6ddcc960-6535-4ae5-8d4b-a80446c608ff">MIIGcAYJKoZIhvcNAQcCoIIGYTCCBl0CAQMxDzANBglghkgBZQMEAgEFADByBgsqhkiG9w0BCRABBKBjBGEwXwIBAQYDKgMEMDEwDQYJYIZIAWUDBAIBBQAEIOWnPp2InqIEp/lbyG2bWMqvRCn6095BOuZZUDZx4a4ZAhEA16I+8lEiJxPX2eCr+PDlixgPMjAyMDA0MTcxMjIzNDdaoIIDcjCCA24wggJWoAMCAQICAWQwDQYJKoZIhvcNAQELBQAwVTEYMBYGA1UEAwwPc2VsZi1zaWduZWQtdHNhMRkwFwYDVQQKDBBOb3dpbmEgU29sdXRpb25zMREwDwYDVQQLDAhQS0ktVEVTVDELMAkGA1UEBhMCTFUwHhcNMTkwMjI4MDUzMTAxWhcNMjAxMjI5MDUzMTAxWjBVMRgwFgYDVQQDDA9zZWxmLXNpZ25lZC10c2ExGTAXBgNVBAoMEE5vd2luYSBTb2x1dGlvbnMxETAPBgNVBAsMCFBLSS1URVNUMQswCQYDVQQGEwJMVTCCASIwDQYJKoZIhvcNAQEBBQADggEPADCCAQoCggEBAKMqiCLZBdWzCBRR3JA/eFPj3eL6s0lV36bHuONNERHPLrpHqOLm78nteX3zNFOTsNYXlXiIN7mOPeRV3gKCLkwEYlF29V90nPwWyPeUpwBerZEsO+PK3DiV0nUIq+UyEeJo4UEZv2vdaL4RFsrzMtvzMZK61LTR1PD3kiatPe8bfNtEpfCnVURklI8sESQus8zznUN86O62dFtAwmqg470llsqPUFIiVSG16mp4bB0k+J8bXc0HdxcydEdaAS4PEj5Ju0VBuFbF0eHxfIe4lzmhI2dSQlEOFAEEtGtIfnKAFF+ziUL5DPTjBmqNDT/koaSGa+/ZAzDUxHIhP1BieWMCAwEAAaNJMEcwDgYDVR0PAQH/BAQDAgeAMBYGA1UdJQEB/wQMMAoGCCsGAQUFBwMIMB0GA1UdDgQWBBSJQivZn9FP3XEbL7CgTM1Km6f+xjANBgkqhkiG9w0BAQsFAAOCAQEAfUnL1vCxvg0rDC3ee/zydpagLZrQZNjg3m9ry575qRVm+9bwpwNCbGDKgrAU4Yv6naRSTPrx0f0P50ELXnCogo9JlJ5fE7SQIVLDyo7mytrtiHZh3IUUv5qGcFka7TumgWpPT6xEo/N55u/ZgN61JavIt7twBwIkk9OapUBEPSg8LaBIX7gTUJP/r9YBcrLIBTFPXQV8xu41/H/qD7kkpF9/UaT22WvF+6ogDVDbg0XvEt14uhn7JHj4R5seLwqSU4gLLaLpfj33uMp5qxjj8TIjXfRiHmIiQEdRGd/o3DBXbWQZYTSzIgjBacGLNLnNMCyo12lSjqsYv3Xc0UWMUzGCAlswggJXAgEBMFowVTEYMBYGA1UEAwwPc2VsZi1zaWduZWQtdHNhMRkwFwYDVQQKDBBOb3dpbmEgU29sdXRpb25zMREwDwYDVQQLDAhQS0ktVEVTVDELMAkGA1UEBhMCTFUCAWQwDQYJYIZIAWUDBAIBBQCggdMwGgYJKoZIhvcNAQkDMQ0GCyqGSIb3DQEJEAEEMBwGCSqGSIb3DQEJBTEPFw0yMDA0MTcxMjIzNDdaMC0GCSqGSIb3DQEJNDEgMB4wDQYJYIZIAWUDBAIBBQChDQYJKoZIhvcNAQELBQAwLwYJKoZIhvcNAQkEMSIEID06zp81/A/HCf3LoiamjHdR8pOAD8zGDJwgnGyQ4uB1MDcGCyqGSIb3DQEJEAIvMSgwJjAkMCIEIG//oLlmXLoxyN1fu5cKUEcp3zdWvnHF0ofMlA8GQjjAMA0GCSqGSIb3DQEBCwUABIIBADSAmBCr4Ui/nH99OwQke3zi1pTAwPA4RSs/hZg+svZkpBynKTFXH/MNDXQBcYV/MOVspduQnIfoo97PvKb4lmBG4O/QoVg3rQnw0NLTmo7jPhofJ6sCUZr3NYaQpwmJwv26c4zsMylefDB4AdaIjHd6n5hmA1gbM8N+38+a9zrmpGyXWJ91OeWxpvIBEuKblg7tp4FKL1ptgijduXzfLRi6ouiw72gVW3k6ctLkGmXBZc5r+eiZGwRl2E5jogglZrwU41dQaL25YhJ7Kc+d1P3BcuiAQlDw1Rz0C6UrGpLnwRkhKlU/qBhm0qFmTIKIhSDckTgEOBf5nGkJXR8KqIk=</xades:EncapsulatedTimeStamp></xades:SignatureTimeStamp></xades:UnsignedSignatureProperties></xades:UnsignedProperties></xades:QualifyingProperties></ds:Object></ds:Signature></dcc:digitalCalibrationCertificate>
```

Comment:
Please pretty print this signature, if feasible.  And in any case add a newline so that the file ends in a newline.

Reply:
We tried to to that, but altering the file in any way causes the validation to fail. Even if we just added newlines etc the effect is the same as changing some "real" values in the XML.  I'll look into this once more to check if it's possible to make it prettier.